docs/public/port_detection.md

```diff
@@ -243,6 +244,8 @@ app.listen(8080, () => {
 })
 ```
 
+In case we have an OR operator with an environment variable alizer will return both ports, first the env var and then the default one. Again it will look first locally for env var and if there is none it will check for dockerfile.
```

Comment:
```suggestion
In case we have an OR operator with an environment variable alizer will return both ports, first the env var and then the default one. Again, it will look first locally for env var and if there is none it will check for dockerfile.
```